cmd/outline-ss-server/main.go

```diff
@@ -297,6 +340,55 @@ func (s *OutlineServer) runConfig(config Config) (func() error, error) {
 						go service.PacketServe(pc, func(ctx context.Context, conn net.Conn) {
 							associationHandler.HandleAssociation(ctx, conn, s.serviceMetrics.AddOpenUDPAssociation(conn))
 						}, s.serverMetrics)
+					} else if cfg.WebsocketStream != nil {
+						if _, exists := webServers[cfg.WebsocketStream.WebServer]; !exists {
+							return fmt.Errorf("listener references unknown web server `%s`", cfg.WebsocketStream.WebServer)
```

Comment:
This will make it easier to pinpoint the error:
```suggestion
							return fmt.Errorf("websocket-stream listener references unknown web server `%s`", cfg.WebsocketStream.WebServer)
```